pymrio/tools/iomath.py

```diff
@@ -19,6 +19,8 @@
 def calc_x(Z, Y):
     """Calculate the industry output x from the Z and Y matrix
 
+    industry output x = flows(Z) + final demand(Y)
```

Comment:
Thats not fully correct: it is: sum_columns(Z) +  sum_columns(Y)

Reply:
ok, corrected in the last commit